website/templates/project/contributors.mako

```diff
@@ -124,7 +124,7 @@
                     <tr>
                         <td class="col-sm-3">
                             <div>
-                                <span class="link-name m-b-xs" data-bind="text: name, tooltip: {title: 'Link name'}" style="display: block; width: 100%"></span>
+                                <span class="link-name m-b-xs" data-bind="html: name, tooltip: {title: 'Link name'}" style="display: block; width: 100%"></span>
```

Comment:
Why are you binding to HTML? If unescaped text does make it into database somehow, this guarantees that it will be inserted directly into the page and treated as code. That could be a potential security vulnerability.